MediaBrowser.Controller/Lyrics/Metadata.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// Metadata model.
+/// </summary>
+public class Metadata
```

Comment:
```suggestion
public class LyricMetadata
```

Use unique and specific names for API models.